index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link href="https://fonts.googleapis.com/css?family=Lato:100,300,400,700,900&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="assets/css/style.css">
+    <title>Apple</title>
+  </head>
+
+  <body>
+    <!-- HEADER -->
+    <!-- Contains the navigation, hero-section with main header and related content -->
+    <!-- background will be achieved using css background properties -->
+    <header class="header">
+      <nav class="nav">
+        <div class="nav__container">
+          <ul class="nav__list">
+            <li><a href="#"><img class="logo" src="assets/img/apple.png" alt="Apple logo"></a></li>
+            <li><a href="#">Store</a></li>
+            <li><a href="#">Mac</a></li>
+            <li><a href="#">iPod</a></li>
+            <li><a href="#">iPhone</a></li>
+            <li><a href="#">iPad</a></li>
+            <li><a href="#">iTunes</a></li>
+            <li><a href="#">Support</a></li>
+            <li><input type="search" name="search" id="search"><img class="search" src="assets/img/search.png"
+                alt="Search icon"></li>
+          </ul>
+        </div>
+      </nav>
+      <h1>What will your verse be?</h1>
```

Comment:
Please follow this bullet points to correct this area of your code
- [ ] There should be three links
- [ ] Each link is an <a> HTML tag, with a <div> tag inside it.
- [ ] The <div> tag  inside the <a> tag should use an image as background -- [ ] The text on the original page is actually an image

Reply:
Dear Raphael,
Thank you for your feedback. I got your feedback and I understand that I need to EXACTLY replicate the reference website. But don't you think these should Heading tags instead of Divs>img ? semantically? 

Looking forward to hear from you.

Reply:
![11](https://user-images.githubusercontent.com/37092867/61338464-7fa69c00-a831-11e9-83d2-f402560cb1a4.png)
That is a screenshot of the actual code on the original side, while you may be right because those look like headings, they are not text, they are actually images, the point is it is better you work with the images, so you can master it.

Reply:
Alright. Thanks a lot for your suggestions. I will update the code based on your advice and send it for review again. Thanks :)

Reply:
Hi, you have not changed the text to images, inspect the original site and get those images then add them to your html, replace those text. That is the h1 and h2 text you have below the nav tag. Then re-submit for review. if you are not clear on this, please message me on slack.